src/components/pen-tests.vue

```diff
@@ -131,14 +131,12 @@
               <div class="d-flex justify-content-center align-items-center">
                 <small class="d-flex justify-content-center flex-column">
                   <span><i class="fa-solid fa-check text-success" /> <a href="https://owasp.org/www-project-application-security-verification-standard/" target="_blank" rel="noopener">OWASP ASVS</a></span>
-                  <span><i class="fa-solid fa-check text-success" /> Custom tailored methodologies</span>
                 </small>
               </div>
             </td>
             <td><success /></td>
             <td><success /></td>
-            <td class="text-success">Retests included and can be used within 12 months</td>
-            <td class="text-success">All senior team and thorough</td>
+            <td class="text-success">Free retests within 12 months</td>
```

Comment:
@wparad The retests are included in the original pricing, I'm not sure saying they're free is the right message. I don't want anybody reading this to get the wrong impression and send you a correction being upset!

Reply:
Honestly I'm not really sure what the difference is here, I just wanted to keep the language consistent with the other places we talk about this.

Realistically "free", always means "priced in", so whether or not it is extra or not isn't necessarily relevant. If anything I'm most concerned about the language "can be used within 12 months", that part felt the most not-obvious while reading. My expectation is that everyone understands what free means here, are you saying that you explicitly have a line item in the contract that says "retests - $K" in the invoice?